Makefile

```diff
@@ -7,7 +7,7 @@ PKG_CONFIG=pkg-config
 CFLAGS+=$(shell $(PKG_CONFIG) --cflags $(LIBS))
 CFLAGS+=-fPIC -DPIC
 LDLIBS+=$(shell pkg-config --libs $(LIBS))
-LDLIBS+=-lhttp_parser
+LDLIBS+=-lhttp_parser -lolm
```

Comment:
OK, starting to fix this stuff on my 'crypto' branch - still a few things.

Fixed: Can now do:
  make -DMATRIX_NO_E2E
and it avoids it